### DISCUSSION TITLE
Empty profile and unable to play multiplayer in osu!lazer

### DISCUSSION THREAD

**Discussion by IGrigori4I** [ppy/osu#27005] [Q&A] [answered]
Don't know when exactly the problem occured because i was just trying to play multiplayer with my sister as always but this time we found that her account is almost empty like described here: https://github.com/ppy/osu-web/issues/9684 
And everytime we tried to start playing together there was pop-up massage on her screen after which her session of beatmap closed immideately and she was back on the room lobby screen(i attached screenshot of this message and her profile)
I did a little research with issue i provided earlier and tried to google the pop-up massage but...nothing there, i guess. People have similar problem and were saying that it can be partly resolved by itself but there was no exact solution to the problem, so i hope to find it here. Also, i should say that my sister's account relatively new, she plays about 1-2 weeks if it`s matter. I'll appreciate any help here

![photo_2024-02-03_19-22-20](https://github.com/ppy/osu/assets/158763149/3cc57216-2302-4a50-90ff-c7a6ed0f6923)
![photo_2024-02-03_19-22-16](https://github.com/ppy/osu/assets/158763149/ec7af6b0-d274-444e-a2c9-10cbc1bee197)

**Comment by bdach** [comment, marked as answer]
please update to newest release. if you are on newest release already, then you're likely on arch linux or some other distro using unofficial binaries; those are no longer supported.

scores set on prior versions of the game have been archived. some may return if we can find a subset of scores we can safely import without making things unfair, but assume that they will not.
